tests/indices/tier/warm.yaml

```diff
@@ -0,0 +1,38 @@
+$schema: ../../json_schemas/test_story.schema.yaml
+
+description: Test endpoints relevant to tiering of index to warm.
+epilogues:
+  - path: /books
+    method: DELETE
+    status: [200, 404]
+  - path: /games
+    method: DELETE
+    status: [200, 404]
+chapters:
+  - synopsis: Create an indices named `books` and `games` with default settings.
+    path: /{index}
+    method: PUT
+    parameters:
+      index: books,games
+
+  - synopsis: Check if the indices `games` and `books` exists. They should.
+    path: /{index}
+    method: HEAD
+    parameters:
+      index: books,games
+      allow_no_indices: true
+      expand_wildcards: none
+      flat_settings: true
+      include_defaults: true
+      ignore_unavailable: true
+      local: true
+
+  - synopsis: Tier the `books` and `games` indices from hot to warm.
+    version: '>= 2.16'
```

Comment:
Since this tier test is entirely >= 2.16 move this `version` into the top level of the test, and it will be fully skipped.

Reply:
didnt know that, let me update this